server/src/main/java/org/opensearch/action/admin/cluster/configuration/AddVotingConfigExclusionsRequest.java

```diff
@@ -157,7 +157,7 @@ Set<VotingConfigExclusion> resolveVotingConfigExclusions(ClusterState currentSta
         } else {
             assert nodeNames.length >= 1;
             Map<String, DiscoveryNode> existingNodes = StreamSupport.stream(allNodes.spliterator(), false)
-                .collect(Collectors.toMap(DiscoveryNode::getName, Function.identity()));
+                .collect(Collectors.toMap(DiscoveryNode::getName, Function.identity(), (r1, r2) -> r1));
```

Comment:
I would suggest reverting this to ensure there are no duplicates and we are not silently handling it

Reply:
Sure